tests/console/system_prepare.pm

```diff
@@ -21,14 +21,16 @@ sub run {
 
     ensure_serialdev_permissions;
 
-    # Stop packagekit as may lock zypper and fail zypper calls, if it's running
-    systemctl 'stop packagekit.service' unless script_run('systemctl is-active packagekit.service');
+    # Mask packagekit as may lock zypper and fail zypper calls, if it's running
+    systemctl 'stop packagekit.service';
+    systemctl 'mask --now packagekit.service';
     # Installing a minimal system gives a pattern conflicting with anything not minimal
     # Let's uninstall 'the pattern' (no packages affected) in order to be able to install stuff
     script_run 'rpm -qi patterns-openSUSE-minimal_base-conflicts && zypper -n rm patterns-openSUSE-minimal_base-conflicts';
     # Install curl and tar in order to get the test data
     zypper_call 'install curl tar';
-
+    # unmask packagekit service as some images expect this service to be running
```

Comment:
so probably we should also start it?

Reply:
We reboot the system afterward, or in other scenarios run consoletest_setup. So basically it's only for create_hdd scenarios where we shutdown system anyway, hence it will be started on boot.
So, yeah, it's a bit messy, but I didn't want to move cutl and tar back to consoletest_setup =)

Reply:
also fails staging tests
reverting for now and having a closer look afterwards